### PR TITLE
MCD wipe and free

### PR DESCRIPTION
**Pull request by levity**
- also fix the erroneous use of the single-collateral DAI token, replacing it with a token called MDAI (placeholder name)
- also add EtherToken.balance, same as Erc20Token.balance

tagging lots of people as reviewers just to get some eyeballs on this quickly, but feel free to skip if you're busy and someone else is on it

### CODE REVIEW

**Comment by levity** [comment]
tests are passing, i swear 😅 Travis is broken due to one of the repos we depend upon being set as private

**Comment by levity** [comment]
also feel free to suggest an alternative to `MDAI`

File: lib/dai-plugin-mcd/src/ManagedCdp.js [hunk 1]
```diff
@@ -22,7 +21,7 @@ export default class ManagedCdp {
       .get(ServiceRoles.CDP_TYPE)
       .getCdpType(this.currency);
     const { rate } = await cdpType.ilkInfo();
-    const art = Maker.DAI.wei((await this._urnInfo()).art);
```
Comment (tyler17) [inline lib/dai-plugin-mcd/src/ManagedCdp.js]: I think art * Ilk.rate is the actual debt value, as @jparklev had mentioned before?  I just realized I also need to update the way `getTotalDebt` is calculated in CdpType to multiply Art by Ilk.rate...

Comment (levity) [reply lib/dai-plugin-mcd/src/ManagedCdp.js]: I multiply by rate below

Comment (tyler17) [reply lib/dai-plugin-mcd/src/ManagedCdp.js]: oops - guess I was projecting haha

**Comment by jparklev** [comment]
LGTM. Is there any reason you went with MDAI over MCD?

**Comment by levity** [comment]
> LGTM. Is there any reason you went with MDAI over MCD?

Just because we already use MCD as a prefix for a lot of contract addresses... I guess that could work too

**Pull request merged**